### ISSUE TITLE
Remove react widget from Board Appeals page and add form page to left nav

### ISSUE THREAD

**Issue by RLHecht** [department-of-veterans-affairs/va.gov-team#40656]
## Issue Description

We put a react widget on the [Board Appeals](https://www.va.gov/decision-reviews/board-appeal/) page for the NOD online form, which is rolling out in a phased launch. Once the form is available to 100% of users, we need to remove the react widget and put the content and the action link into Drupal. Robin Garrison said this should be June 8, 2022, but we will confirm as it gets closer. 

Here is the text that will need to be added to the Drupal page: [https://app.zenhub.com/files/133843125/d7724eca-ff6b-416a-ad81-3953120429d5/download](https://app.zenhub.com/files/133843125/d7724eca-ff6b-416a-ad81-3953120429d5/download)

We also need to add the online form page to the decision reviews left nav. It should be the first child page of Board Appeals. 

Text for left nav: Request a Board Appeal
url: https://www.va.gov/decision-reviews/board-appeal/request-board-appeal-form-10182

## Tasks
- [x] Confirm date to remove react widget (Randi)
- [x] Remove react widget 
- [x] Add content in Word doc above into Drupal page
- [x] Review and publish (Megan)
- [x] Add form page to left nav (Megan)

## Acceptance Criteria
- [x] All content on live page is in Drupal and react widget is no longer on page.
- [x] Online form showing in decision reviews left nav

**Comment by bethpottsVADEPO** [comment]
@RLHecht When it comes time to do this work, it'll probably be good to train at least @laurwill and @megzehn on this process (including left nav updates). Do you think it would be helpful for the whole team to know how to do this or is it sufficient for the 3 of you to be the ones who make these updates? Would you talk as a team about this and let me know? Thanks!

**Comment by RLHecht** [comment]
Update: I've removed the react widget and added the approved content back into Drupal. @megzehn This is ready for you to do a copy edit but we can't publish until we get the go-ahead from the product team that the tool is launched to 100% (I also noted that in the revision log). When we're ready to publish we can walk through the left nav process again. Here's the documentation as well: https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/vsa/teams/sitewide-content/how-to-do-different-tasks/adding-left-nav-and-url.md

**Comment by megzehn** [comment]
Thanks, @RLHecht! I did an initial copy edit, but I have a couple of questions about updating the left nav and alert box that I'm hoping to sync with you on. 

On a related note: how finalized is this copy? I made a slight text change that I wanted to flag. Under “What are my Board Appeal options?” I added “If you choose the … option,” to the first two, bc it was hard to follow without. But if the text needs to remain the same, lmk, and I will roll them back.

**Comment by RLHecht** [comment]
@megzehn I added the form page to the left nav. It was actually already there but disabled because I had put it in a while ago when we thought the form going to 100% initially. So I actually just enabled it. You'll still need to publish the Board Appeals page though. Thank you!

**Comment by RLHecht** [comment]
Confirmed that the form is now showing in the left nav. FYSA @mnorthuis Megan, you'll still need to publish the Board Appeals page when you get the chance.

![Screen Shot 2022-06-10 at 7.23.47 AM.png](https://images.zenhubusercontent.com/5c9b7e302ac54336e2c327b8/5028acd0-3c85-4c94-ad01-07fe583e6aad)

**Comment by megzehn** [comment]
Published!

**Comment by RLHecht** [comment]
@megzehn I think the Board Appeals page still needs to be published. Thanks!

**Comment by megzehn** [comment]
sorry @RLHecht. Could you check again?

**Comment by RLHecht** [comment]
All good! Closing this ticket.
